internal/services/notificationhub/notification_hub_namespace_resource.go

```diff
@@ -216,6 +240,8 @@ func resourceNotificationHubNamespaceRead(d *pluginsdk.ResourceData, meta interf
 		if props := model.Properties; props != nil {
 			d.Set("enabled", props.Enabled)
 			d.Set("servicebus_endpoint", props.ServiceBusEndpoint)
+			d.Set("zone_redundancy_enabled", pointer.From(props.ZoneRedundancy) == namespaces.ZoneRedundancyPreferenceEnabled)
+			d.Set("replication_region", string(pointer.From(props.ReplicationRegion)))
```

Comment:
nit: there's a new(er) helper we can use here

```suggestion
			d.Set("replication_region", pointer.FromEnum(props.ReplicationRegion)))
```